cmd/lakefs/cmd/root.go

```diff
@@ -41,26 +43,50 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "", "config file (default is $HOME/.lakefs.yaml)")
 	rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
 	rootCmd.PersistentFlags().Bool(config.UseLocalConfiguration, false, "Use lakeFS local default configuration")
+	rootCmd.PersistentFlags().Bool(config.QuickStartConfiguration, false, "Use lakeFS quickstart configuration")
 }
 
-func useLocal() bool {
-	res, err := rootCmd.PersistentFlags().GetBool(config.UseLocalConfiguration)
+func validateQuickstartEnv(cfg *config.Config) {
+	if cfg.Database.Type != local.DriverName || cfg.Blockstore.Type != block.BlockstoreTypeLocal {
+		fmt.Printf("quickstart mode can only run with local settings\n")
+		os.Exit(1)
+	}
+	accessKey := "AKIAIOSFOLQUICKSTART"
+	secretKey := "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY"
+	cfg.Installation.UserName = "quickstart"
+	cfg.Installation.AccessKeyID = config.SecureString(accessKey)
+	cfg.Installation.SecretAccessKey = config.SecureString(secretKey)
+	fmt.Printf("Access Key ID    : %s \n", accessKey)
+	fmt.Printf("Secret Access Key: %s \n", secretKey)
+}
+
+func useConfig(cfg string) bool {
+	res, err := rootCmd.PersistentFlags().GetBool(cfg)
 	if err != nil {
-		fmt.Printf("%s: %s\n", config.UseLocalConfiguration, err)
+		fmt.Printf("%s: %s\n", cfg, err)
 		os.Exit(1)
 	}
 	if res {
-		printLocalWarning(os.Stderr, "local parameters configuration")
+		printLocalWarning(os.Stderr, fmt.Sprintf("%s parameters configuration", cfg))
 	}
 	return res
 }
 
 func newConfig() (*config.Config, error) {
-	if useLocal() {
-		return config.NewLocalConfig()
-	}
+	quickStart := useConfig(config.QuickStartConfiguration)
```

Comment:
Do we need to verify that not both are on?

Reply:
Not sure we wan't to enforce mutual inclusion - do you see an issue with it?